### ISSUE TITLE
Config example for Federated User Registry

### ISSUE THREAD

**Issue by dmuelle** [OpenLiberty/docs#1180]
Incorporating examples for federating basic and custom user registries. Since LDAP are configured by default, LDAP examples are in the ldapRegistry feature

**Comment by dmuelle** [comment]
@lauracowen draft at https://draft-openlibertyio.mybluemix.net/docs/ref/feature/#federatedRegistry-1.0.html

**Comment by lauracowen** [comment]
Thanks

- in the first config example, can you remove the unnecessary parts of the example? The text above says to enable both features (with the link to the App Security feature) so that's fine, you don't need to show that in the example. So include only the two basicRegistry sections but remove from those as much as possible (ie remove unnecessary groups and details of the groups) to keep the example as short as possible. Examples of how to use basicRegistry (eg groups, multiple groups, multiple users) should be shown in the appSecurity feature (https://draft-openlibertyio.mybluemix.net/docs/ref/feature/#appSecurity-3.0.html). Assume here that they know how to set up basicRegistry for the configuration that they need. This example should show only how to do federation of the basic registries - otherwise it becomes confusing about what parts of the example are relevant/necessary to federation.
- otherwise, it all looks good, thanks :)

**Comment by dmuelle** [comment]
@lauracowen I think this one might be ready to publish as well?

**Comment by lauracowen** [comment]
Yep, I agree. Thanks

**Comment by jvanhill** [comment]
Developer approved.

**Comment by ManasiGandhi** [comment]
#peer review
@dmuelle The topic looks good. I just have a few edit suggestions.

- In the first paragraph under 'Auto federation of user registries', the second sentence says,"To federate two or more user registries, enable both the Application Security feature and the Federated User Registry feature and the registries federate by default". Can you rephrase this to "To federate two or more user registries, enable both the Application Security feature and the Federated User Registry feature, and the registries federate by default."
- Similarly, in the first sentence of the second paragraph, "Any user registries, including basic, LDAP, and custom registries, that are defined in the same server configuration federate by default.", can you add a comma after configuration to, "Any user registries, including basic, LDAP, and custom registries, that are defined in the same server configuration, federate by default. "
- In the first sentence under 'Custom federation of user registries', "If you want to specify a subset of user registries to federate into custom federated repository, you can define the participatingBaseEntry name attribute in the federatedRepositoryelement.", maybe add 'a' before 'custom federated repository', so that it reads, "If you want to specify a subset of user registries to federate into a custom federated repository, you can define the participatingBaseEntry name attribute in the federatedRepository element. "

**Comment by dmuelle** [comment]
Made the three edits that @ManasiGandhi  recommended above.

**Comment by dmuelle** [comment]
new draft URL: https://draft-openlibertyio.mybluemix.net/docs/20.0.0.11/reference/feature/federatedRegistry-1.0.html

**Comment by dmuelle** [comment]
edits made in new pass:

- Auto federation of user registries --> Default federation of user registries DQTI Ch7 concreteness: Use specific language
- If you want to specify a subset of user registries to federate into a custom federated repository...-->  If you want to federate just a subset of user registries into a custom federated repository DQTI ch6 Clarity: Focus on the meaning

**Comment by chirp1** [comment]
@dmuelle Hi!
Here are my comments:
- The topic is nice, but make information among your topics accurate, clear, and consistent. You have information on default repositories that isn't clear and is inconsistent with info in the "User registries for application security" topic.  The following bullets have some of the details:
- As mentioned in the peer review cheat sheet, be consistent with your terms in a given topic and across topics. Sometimes the topics say, for example " custom user registry" and other times "custom registry".
- ", and the registries federate by default. " is oddly tacked on to the end of a sentence. I don't think this wording is needed as the next paragraph discussed default federation.
- You mention "userRegistry.xml " in this topic but not in your "User registries for application security" topic.
- In "Any user registries that are defined in the same server configuration federate by default...", what do you mean by "server configuration"? Do you mean the server.xml file or do you mean both the server.xml file and the userRegistry.xml  file? Specify the files that you mean, and be consistent with your "User registries for application security" topic, which says that only user registries in the server.xml file are federated by default. 
- In this topic you say that basic, LDAP, and custom user registries are federated by default, but in the  "User registries for application security" topic you say that only basic and LDAP user registries are federated by default.
- Change "federate just a subset " to "federate a subset ". <= get rid of the extra word.
- Don't put parentheses around the term participatingBaseEntry . If you specify the term, put a noun or noun phrase after the term. Note that a KC topic gives a noun phrase for the term.

**Comment by dmuelle** [comment]
@chirp1 thanks for reviewing, I made the following changes per your review:

- [x]  be consistent with your terms in a given topic and across topics. Sometimes the topics say, for example " custom user registry" and other times "custom registry". **fixed**

- [x]  ", and the registries federate by default. " is oddly tacked on to the end of a sentence. **fixed**

- [x] You mention "userRegistry.xml " in this topic but not in your "User registries for application security" topic **added mention of userRegistry.xml to User Registry topic**

- [x]  In "Any user registries that are defined in the same server configuration federate by default...", what do you mean by "server configuration" **clarified the relevant files**

- [x] In this topic you say that basic, LDAP, and custom user registries are federated by default, but in the  "User registries for application security" topic you say that only basic and LDAP user registries are federated by default. **fixed in the user registry topic**

- [x] Change "federate just a subset " to "federate a subset ". <= get rid of the extra word.

- [x] Don't put parentheses around the term participatingBaseEntry 

https://draft-openlibertyio.mybluemix.net/docs/20.0.0.11/reference/feature/federatedRegistry-1.0.html

**Comment by chirp1** [comment]
@dmuelle Hi David, the topic looks good! I'm moving it to "Ready to Publish".

**Comment by dmuelle** [comment]
moved to `vNext` to publish with 20.0.0.11. Closing this issue.
